priv/repo/seeds.exs

```diff
@@ -9,3 +9,66 @@
 #
 # We recommend using the bang functions (`insert!`, `update!`
 # and so on) as they will fail if something goes wrong.
+
+execution_id = "fa5f4440-14ee-4169-8723-7cc7660f8024"
+group_id = "e71961b1-6c94-4b0a-84fb-7e00cc614d9d"
```

Comment:
Is it necessary that these are fixed? Could we just generate them?

Reply:
Actually, why do we need any seed here? I would completely remove this

Reply:
Well, if the choice was between this and the photofinish fixture... :smile: maybe we can remove it in a second moment? At the moment it does no harm and it allows to get started with some work that implements the read side of this

Reply:
I think we need seeds to test the API, right?

Reply:
Sorry missed your comments @nelsonkopliku, yeah maybe in the next PR you can move it in a test

Reply:
> I think we need seeds to test the API, right?

Yes, it might be useful sometimes to have wanda in dev mode filled with data.

Reply:
00000000-....1 is a valid uuid

Reply:
```elixir
execution_id = "00000000-0000-0000-0000-000000000001"
group_id = "00000000-0000-0000-0000-000000000002"
```